spring-boot-security/src/main/java/com/baeldung/springbootsecurity/basic_auth/config/BasicAuthConfiguration.java

```diff
@@ -14,25 +14,27 @@
 
     @Override
     protected void configure(AuthenticationManagerBuilder auth) throws Exception {
-    	PasswordEncoder encoder = PasswordEncoderFactories.createDelegatingPasswordEncoder();
-    	auth
-          .inMemoryAuthentication()
-          .withUser("user")
-          .password(encoder.encode("password"))
-          .roles("USER")
-          .and()
-          .withUser("admin")
-          .password(encoder.encode("admin"))
-          .roles("USER", "ADMIN");
+        PasswordEncoder encoder =
+                PasswordEncoderFactories.createDelegatingPasswordEncoder();
```

Comment:
This should be a 2-space indent

Reply:
you're right. When I reformat the page, it seemed ok for the Intellij. Now I fix it.